openmc/filter.py

```diff
@@ -531,12 +531,13 @@ def get_pandas_dataframe(self, data_size, summary=None):
         ----------
         data_size : Integral
             The total number of bins in the tally corresponding to this filter
-        summary : None or openmc.Summary
-            An optional Summary object to be used to construct columns for
-            distribcell tally filters (default is None). The geometric
+        distribcell_paths : bool
```

Comment:
Put ", optional" at the end of this line and mention the default in the description. Also, is there a particular reason we shouldn't use the distribcell paths by default?